ci/travis-centos-rhel8.sh

```diff
@@ -95,38 +92,27 @@ for phase in "${PHASES[@]}"; do
                         -dit --net=host centos:$CENTOS_RELEASE /sbin/init
             # Beautiful workaround for Fedora's version of Docker
             sleep 1
-            $DOCKER_EXEC yum makecache
-            $DOCKER_EXEC curl "$COPR_REPO" -o "$COPR_REPO_PATH"
-            $DOCKER_EXEC yum -q -y install epel-release yum-utils
-            $DOCKER_EXEC yum-config-manager -q --enable epel
-            $DOCKER_EXEC yum -y upgrade
-            # Install necessary build/test requirements
-            $DOCKER_EXEC yum -y install "${ADDITIONAL_DEPS[@]}"
-            $DOCKER_EXEC python3.6 -m ensurepip
-            $DOCKER_EXEC python3.6 -m pip install meson
-            # Create necessary symlinks
-            $DOCKER_EXEC ln --force -s /usr/bin/python3.6 /usr/bin/python3
-            $DOCKER_EXEC ln --force -s /usr/bin/ninja-build /usr/bin/ninja
+            $DOCKER_EXEC dnf makecache
+            # Install and enable EPEL
+            $DOCKER_EXEC dnf -q -y install epel-release dnf-utils "${ADDITIONAL_DEPS[@]}"
+            $DOCKER_EXEC dnf config-manager -q --enable epel
+            # Upgrade the container to get the most recent environment
+            $DOCKER_EXEC dnf -y upgrade
+            # Install systemd's build dependencies
+            $DOCKER_EXEC dnf -q -y --enablerepo "PowerTools" builddep systemd
             ;;
         RUN)
             info "Run phase"
             # Build systemd
-            docker exec -it -e CFLAGS='-g -O0 -ftrapv' $CONT_NAME meson build "${CONFIGURE_OPTS[@]}"
+            docker exec -it -e CFLAGS='-g -O1 -ftrapv' $CONT_NAME meson build "${CONFIGURE_OPTS[@]}"
```

Comment:
I don't think that the -O0 should be changed. IMHO the tests should be run either with -O0, or the same options as we use in the spec. Maybe even both, but that's probably out of scope for this PR.

Is there a reason for the -O1 change?

Reply:
That was my mistake, the `-O1` has to be in the ASan/UBSan job due to https://github.com/systemd/systemd/issues/12997, will fix it asap.

Reply:
Ah, my past me already tried to do that, but failed, see https://github.com/systemd/systemd-centos-ci/pull/144#issuecomment-510182334. I guess I'll just revert the -Ox change, as we have gcc 8.x in RHEL8, which doesn't trigger this issue, and be done with it.